index_test.go

```diff
@@ -202,7 +202,7 @@ func (s *memSeeker) Size() (uint32, error) {
 func TestNewlines(t *testing.T) {
 	b := testIndexBuilder(t, nil,
 		Document{Name: "filename", Content: []byte("line1\nline2\nbla")})
-	// ---------------------------------------------012345-678901-234
+	// -------------------------------------------012345-678901-234
```

Comment:
not sure if just github display, but this doesn't align before or after. I would suspect the 0 needs to line up with the very first char of the string.

Reply:
Oh yep. My editor's tabwidth made made this not line up because the comment didn't start at the same indent level. Moved the comment above the line which should make it look right for everyone